ocs_ci/utility/rosa.py

```
@@ -61,6 +61,84 @@ def login():
+    """
+    aws = AWSUtil()
+    removed = 0
+    active_clusters = set()
+    try:
+        out = utils.exec_cmd(
+            "ocm get /api/clusters_mgmt/v1/clusters --parameter size=200",
+            timeout=120,
+        )
+        clusters_data = json.loads(out.stdout.decode())
+        active_clusters = {c["id"] for c in clusters_data.get("items", [])}
+        logger.info(f"Found {len(active_clusters)} active clusters in OCM")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)